src/windows/export.py

```diff
@@ -370,30 +381,36 @@ def cboSimpleProjectType_index_changed(self, widget, index):
         # parse the xml files and get targets that match the project type
         project_types = []
         acceleration_types = {}
-        for preset_path in [info.EXPORT_PRESETS_PATH, info.USER_PRESETS_PATH]:
-            for file in os.listdir(preset_path):
-                xmldoc = xml.parse(os.path.join(preset_path, file))
-                type = xmldoc.getElementsByTagName("type")
-
-                if _(type[0].childNodes[0].data) == selected_project:
-                    titles = xmldoc.getElementsByTagName("title")
-                    videocodecs = xmldoc.getElementsByTagName("videocodec")
-                    for title in titles:
-                        project_types.append(_(title.childNodes[0].data))
-                    for codec in videocodecs:
-                        codec_text = codec.childNodes[0].data
-                        if "vaapi" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-vaapi.svg")
-                        elif "nvenc" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-nvenc.svg")
-                        elif "dxva2" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-dx.svg")
-                        elif "videotoolbox" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-vtb.svg")
-                        elif "qsv" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-qsv.svg")
-                        elif openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-none.svg")
+        for preset_folder in [info.EXPORT_PRESETS_PATH, info.USER_PRESETS_PATH]:
+            for file in os.listdir(preset_folder):
+                preset_path = os.path.join(preset_folder, file)
+                try:
+                    xmldoc = xml.parse(preset_path)
+                    type = xmldoc.getElementsByTagName("type")
+
+                    if _(type[0].childNodes[0].data) == selected_project:
+                        titles = xmldoc.getElementsByTagName("title")
+                        videocodecs = xmldoc.getElementsByTagName("videocodec")
```

Comment:
Same thing here, re: nesting
```suggestion
                try:
                    xmldoc = xml.parse(preset_path)
                    type = xmldoc.getElementsByTagName("type")

                    if _(type[0].childNodes[0].data) != selected_project:
                        continue

                    titles = xmldoc.getElementsByTagName("title")
                    videocodecs = xmldoc.getElementsByTagName("videocodec")
```